parameters.yaml

```diff
@@ -6,3 +6,12 @@ parameters:
     required: true
     type: string
     description: Cluster ID
+
+  OrganizationIdPathParameter:
+    name: organization_id
+    in: path
+    required: true
+    type: string
+    description: |
+        An ID for the organization to be created is the only required parameter.
+        This ID can only contain alpha-numeric characters, hyphens, underscores, and dots.
```

Comment:
<img width="1032" alt="screen shot 2017-05-24 at 13 42 31" src="https://cloud.githubusercontent.com/assets/455309/26401572/e0bf9524-4086-11e7-8053-b41b9158bd73.png">

You can see that apparently hyphens, dots, and even colons are ok